### PR TITLE
making search requests more user friendly

### PR DESCRIPTION
**Pull request by Jayashree-panda**
#62 making search requests more user friendly
https://drive.google.com/file/d/1dJm0LVeNoYqE0rR4l1zupx1vGxq9EwG7/view

### CODE REVIEW

**Comment by Harkishen-Singh** [comment]
@Jayashree-panda  please resolve conflicts!

**Comment by Jayashree-panda** [comment]
@muskankhedia yup!! I am applying the conditions.

File: service/controllers/messages.go [hunk 1]
```diff
@@ -64,26 +64,56 @@ func routes(routeObject response, w http.ResponseWriter) {
 
 	message := routeObject.message
 	messageArr := strings.Split(message, " ")
-	// messageTemp := message
 	var firstPars string
-	if strings.Contains(message, " ") {
+	var secondPars string
+	var thirdPars string
+	if(len(messageArr)>=3){
```
Comment (Harkishen-Singh) [inline service/controllers/messages.go]: please maintain the linting standards! avoid using () in **if, for, switch** cases and maintain a defined spacing in each statement

File: service/controllers/messages.go [hunk 2]
```diff
@@ -64,26 +64,56 @@ func routes(routeObject response, w http.ResponseWriter) {
...
+		firstPars=messageArr[0]
+		secondPars = messageArr[1]
+		thirdPars = messageArr[2]
+	}else if(len(messageArr)==2){
```
Comment (Harkishen-Singh) [inline service/controllers/messages.go]: same

File: service/controllers/messages.go [hunk 3]
```diff
@@ -146,8 +176,8 @@ func routes(routeObject response, w http.ResponseWriter) {
 		w.Write(jData)
 		TextToSpeech(responseJSON.Message, 0)
 
-	} else if strings.ToLower(firstPars) == "weather" {
-
+	} else if strings.ToLower(firstPars) == "weather" || strings.ToLower(firstPars) == "see" || strings.ToLower(secondPars)=="at"||strings.ToLower(secondPars) == "weather"||strings.ToLower(thirdPars) == "at"{
```
Comment (Harkishen-Singh) [inline service/controllers/messages.go]: **see, at** ??

File: service/controllers/messages.go [hunk 4]
```diff
@@ -130,8 +160,8 @@ func routes(routeObject response, w http.ResponseWriter) {
 		w.Write(jData)
 		TextToSpeech(responseJSON.Message, 0)
 
-	} else if strings.ToLower(firstPars) == "youtube" {
-		query := "https://www.youtube.com/results?search_query=" + messageExceptFirstPars
+	} else if strings.ToLower(firstPars) == "youtube" || strings.ToLower(firstPars)=="video" || strings.ToLower(firstPars) == "watch" || strings.ToLower(secondPars)=="youtube"{
```
Comment (Harkishen-Singh) [inline service/controllers/messages.go]: change to **videos** from **video**

**Comment by Harkishen-Singh** [comment]
overall, the functionality of the PR looks **clean** to me!
Still, attach a **video** explaining the working of code!

File: service/controllers/messages.go (hunk 3 above)
Comment (Jayashree-panda) [reply service/controllers/messages.go]: see weather 
or 
see weather at 
shall i keep it or discard it?

Comment (Jayashree-panda) [reply service/controllers/messages.go]: I am discarding it. It's of no use. And I am changing image to images

**Pull request closed without merging**